### ISSUE TITLE
Any interest in futures.rs domain?

### ISSUE THREAD

**Issue by benesch** [rust-lang-nursery/futures-rs#1488]
There have now been multiple occasions where I've expected futures.rs to land me at the [futures-rs homepage](https://rust-lang-nursery.github.io/futures-rs/). Since the the domain was available I just decided to buy it and point it to the docs. If y'all are interested in making futures.rs the official home of the docs I'd be happy to adjust DNS records or transfer ownership. (Otherwise I'm happy just leaving it as an unusually expensive bookmark.)

**Comment by Nemo157** [comment]
Over the last couple of days I've been using this to get to the docs (instead of having to manually type out `rust-lang-nursery.github.io/futur` till auto-complete works), it's been super useful. So thanks a lot for setting this up :heart:

I definitely think it would be great to make it official and change the GitHub Pages CNAME to use it as the canonical domain, but I'm not sure who would need to be involved to make that happen (@aturon maybe?).

**Comment by thomaseizinger** [comment]
Note that the `futures.rs` domain no longer points at the docs but at a completely different website now so I think this can be closed as no longer actionable.

**Comment by benesch** [comment]
Yes, it didn't seem like there was interest so I let the registration lapse.
